sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -873,6 +873,41 @@ class SQLAppStatusListenerSuite extends SharedSparkSession with JsonTestUtils
     assert(metrics(innerMetric.id) === expectedInnerValue)
   }
 
+  test("Report driver metrics from Datasource v2 scan") {
```

Comment:
Could you add the JIRA number? I.e. "SPARK-39635: Report driver metrics ..."

Reply:
done